src/main/resources/application-dev.yml

```
@@ -3,11 +3,9 @@ spring:
     name: jova-server-v2
   datasource:
     driver-class-name: com.mysql.cj.jdbc.Driver
-    url: jdbc:mysql://${RDB_HOST:localhost}:${RDB_PORT:3306}/${RDB_NAME:jova-dev}?serverTimezone=Asia/Seoul
+    url: jdbc:mysql://${RDB_HOST:localhost}:${RDB_PORT:3306}/${RDB_NAME:jova-dev}?useSSL=false&serverTimezone=UTC
     username: ${RDB_USERNAME:root}
     password: ${RDB_PASSWORD:root}
   jpa:
     generate-ddl: on
-    hibernate:
-      ddl-auto: update
     show-sql: true
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)